doc/source/user_guide/quantity.rst

```diff
@@ -63,6 +59,20 @@ insert them within an equation to perform mathematical operations.
     m_dv = meter / 2  # 0.5
     m_sq = meter**2  # 1
 
+Additions and subtraction between two ``Quantity`` objects retains the units
```

Comment:
What if it's more than two and with different units.
```
inch - meter + foot + km - micrometer
"""

Reply:
For now inch would win.